docs/apidiffs/current_vs_latest/opentelemetry-exporter-zipkin.txt

```diff
@@ -1,4 +1,10 @@
 Comparing source compatibility of  against 
++++  NEW CLASS: PUBLIC(+) FINAL(+) io.opentelemetry.exporter.zipkin.OtelToZipkinSpanTransformer  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.exporter.zipkin.OtelToZipkinSpanTransformer create()
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.exporter.zipkin.OtelToZipkinSpanTransformer create(java.util.function.Supplier)
```

Comment:
No need for these to be public anymore. And I believe the `create()` method is unused.

Reply:
Hiding `create(Supplier)` restricts the user's ability to customize how the IP address is obtained, which, as previously discussed, is a big part of this changeset. The goals are:

1) Allow users to suppress inclusion of the IP address on the zipkin local endpoint (part of the zipkin span).
2. Allow a future change to provide a `Supplier` that is able to detect changes in local IP address and return updated values (probably with caching).

Not sure what we should do about this then. If you want `create()` hidden, it will defeat the purpose.

Reply:
I just meant that the no argument version of create isn't needed, since I don't see any usages. Maybe missing something though.

Reply:
Yeah I think you're right -- it was there as a default, but the builder has its own default and should probably be the source. I'll remove it.